src/components/atoms/HomeArrow.vue

```diff
@@ -1,13 +1,50 @@
 <template>
-	<div class="home__arrow-outline">
+	<div v-on:click="traverse(arrow, containers)" class="home__arrow-outline">
 		<img v-bind:src="arrow.src" v-bind:alt="arrow.alt" class="home__arrow">
 	</div>
 </template>
 
 <script>
 	export default {
 		name: "homeArrow",
-		props: ["arrow"]
+		props: ["arrow", "containers"],
+		methods: {
+			traverse: function(arrow, containers) {
+				console.log(containers);
+				const length = containers.length;
+				
+				let active = 0;
+				for (; active < length-1 ; active++) {
```

Comment:
what this loop even do?

Reply:
Checks which image container has attribute `isHidden == 0`

Reply:
you can use ES6 filter function here, its more readable